### PR TITLE
Add link to community plugins list

### LINKED ISSUE #12431: Not all the community plugins tools are mentioned in the site
The Steps are as follow:
1. Go to https://prettier.io/
2. Scroll down to "Works with the tools you use"

In here we can see that *Community Plugins* doesn't mention all the examples that you see when you search "community plugins" in the search bar. An easy way to fix this is to redirect *Community plugins* to this link https://prettier.io/docs/en/plugins.html#community-plugins which mentions all the plugins used

(just like Yaml and GraphQL are redirected to their websites by clicking on them)

### PR DESCRIPTION
**Pull request by Harry-Hopkinson**
## Description

Redirect community plugins. Fixes #12431

## Checklist

- [x] I’ve added tests to confirm my change works.
- [x] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory).
- [x] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/main/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

**Comment by fisker** [comment]
Can you focus one thing on one PR?

**Comment by Harry-Hopkinson** [comment]
Sure

File: website/data/languages.yml [hunk 1]
```diff
@@ -40,6 +42,7 @@
   image: /images/languages/tools_yaml.svg
   variants: []
 - name: Community Plugins
+  nameLink: https://prettier.io/docs/en/plugins.html#community-plugins
```
Comment (fisker) [inline website/data/languages.yml:45]:
```suggestion
  nameLink: ./docs/en/plugins.html#community-plugins
``

File: website/data/languages.yml [hunk 2]
```diff
@@ -50,3 +53,4 @@
     - "[Ruby](https://github.com/prettier/plugin-ruby)"
     - "[TOML](https://github.com/bd82/toml-tools/tree/master/packages/prettier-plugin-toml)"
     - "[XML](https://github.com/prettier/plugin-xml)"
+    - "[And more...](https://prettier.io/docs/en/plugins.html#community-plugins)"
```
Comment (fisker) [inline website/data/languages.yml:56]:
```suggestion
    - "[And more...](./docs/en/plugins.html#community-plugins)"
```

File: website/data/languages.yml [hunk 3]
```diff
@@ -1,4 +1,5 @@
 - name: JavaScript
+  nameLink: https://www.javascript.com/
```
Comment (fisker) [inline website/data/languages.yml:2]:
```suggestion
```

File: website/data/languages.yml [hunk 4]
```diff
@@ -28,6 +29,7 @@
   variants:
     - "[GraphQL Schemas](https://graphql.org/learn/schema/)"
 - name: Markdown
+  nameLink: "https://www.markdownguide.org/getting-started/"
```
Comment (fisker) [inline website/data/languages.yml:32]:
```suggestion
```

**Review by fisker — CHANGES_REQUESTED** [review]

**Review by fisker — APPROVED** [review]
LGTM, thank you!

**Comment by Harry-Hopkinson** [comment]
Thank you very much 😄

**Comment by fisker** [comment]
To the reviewer who merge this: Please clean up the commit message when merging.

**Review by kachkaev — APPROVED** [review]

**Pull request merged**